libsubmit/providers/aws/aws.py

```diff
@@ -386,13 +388,13 @@ def security_group(self, vpc):
         by changing the allowed port ranges.
 
         :param vpc - VPC in which to set up security group
-        '''
+        """
 
         sg = vpc.create_security_group(
             GroupName="private-subnet", Description="security group for remote executors"
         )
 
-        ip_ranges = [{'CidrIp': '172.32.0.0/16'}]
+        ip_ranges = [{'CidrIp': '10.0.0.0/16'}]
```

Comment:
Not entirely sure why we need this change. But I don't see why this would do any harm.